controllers/usersignup/usersignup_controller_test.go

```diff
@@ -4585,15 +4539,12 @@ func TestUserSignupLastTargetClusterAnnotation(t *testing.T) {
 		spc1 := hspc.NewEnabledValidTenantSPC("member1")
 		spc2 := hspc.NewEnabledValidTenantSPC("member2")
 		r, req, _ := prepareReconcile(t, userSignup.Name, spc1, spc2, userSignup, baseNSTemplateTier, deactivate30Tier, commonconfig.NewToolchainConfigObjWithReset(t, testconfig.AutomaticApproval().Enabled(true)))
-		InitializeCounters(t, NewToolchainStatus())
 
-		// set acceptable capacity for member2 cluster
-		toolchainStatus := &toolchainv1alpha1.ToolchainStatus{}
-		err := r.Client.Get(context.TODO(), types.NamespacedName{Name: toolchainconfig.ToolchainStatusName, Namespace: req.Namespace}, toolchainStatus)
-		require.NoError(t, err)
-		WithMember("member2", WithNodeRoleUsage("worker", 68), WithNodeRoleUsage("master", 65))(toolchainStatus)
-		err = r.Client.Status().Update(context.TODO(), toolchainStatus)
-		require.NoError(t, err)
+		// make the member2 SPC valid so that we simulate that it now has enough capacity
+		member2Spc := &toolchainv1alpha1.SpaceProvisionerConfig{}
+		require.NoError(t, r.Client.Get(context.TODO(), runtimeclient.ObjectKeyFromObject(spc2), member2Spc))
+		spc.ModifySpaceProvisionerConfig(member2Spc, spc.WithReadyConditionValid())
+		require.NoError(t, r.Client.Status().Update(context.TODO(), member2Spc))
```

Comment:
as above - we are not testing that the toolchainstatus capacity is reflected in the SPC, but just that when the SPC is enabled and ready we can provision users in that cluster.

Reply:
Again, the test name was too specific, I renamed it to `last target cluster annotation is set and cluster is ready` to reflect what this tests does.